traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
+
+      {% if not user.is_anonymous %}
+      <div class="post-comment__container">
+        <div class="post-comment__header">
+          <span uk-icon="user"></span>
+          <span>Comment as {{user.get_full_name}}</span>
+          <button class="uk-button button button--primary post-comment__submit" onclick="commentButtonClicked()">Submit</button>
+        </div>
```

Comment:
I feel that the representation of text comments could be made more compact. Currently we have:

![image](https://user-images.githubusercontent.com/10669118/89111402-3b426a00-d44d-11ea-942f-5e63c7ac6ba7.png)

**Kirill Tsukanov commented**  
August 1st 2020, 23:08  
(blank line)  
(Comment text)  
(blank line)

This can be shortened to:

**Kirill Tsukanov** commented on **August 1st 2020, 23:08**  
(Comment text)